### DISCUSSION TITLE
Could you do some tutorials, pls!?  (Especially 'Import'-function)

### DISCUSSION THREAD

**Discussion by MikaNoPlan** [ImranR98/Obtainium#637] [Q&A]
Hi !

Sorry, but could you do some tutorials, pls !? 
Especially for the 'Import'-function !? 
So that 'No-geeks' (with highest respect!) could understand and also easier use your App...
(Or link them somehow, if there are some, because i couldn't find them...
Because i really would like to use and understand it !)

(Sorry, but english isn't my Language and i am new here. So if my request is inappropriate or at the wrong place...  just sorry!)

Best wishes,  Mika

**Comment by DwainZwerg** [comment]
What is your problem with the import function?

**Comment by ImranR98** [comment]
There are no tutorials right now, but this video does have a basic usage overview: https://youtu.be/JiN37bn0OE8

Import/export is simple: "Export" places a file in your downloads folder - this file contains all Obtainium apps (not settings). "Import" lets you pick an export file to re-add its contents into Obtainium.

**Comment by ImranR98** [comment]
I've added a wiki section - it's mostly empty right now though. I'll add to it over time - any contributions are welcome!
